content/using/install.md

```diff
@@ -1,246 +1,212 @@
 +++
-title = "Install + Setup"
+title = "Getting Started"
 weight = 1
 description = "How to install Urbit."
 template = "page_indiced.html"
 aliases = ["/docs/getting-started/"]
 +++
 
-This guide covers how to install and set up your own planet (or comet), but installation for a star is exactly the same. You will first install the Urbit binary, then you will either boot a comet or a planet.  It's not too difficult, but if you would prefer to have someone run your urbit for you, consider a [hosting provider](#hosting-providers).
+The price of using other platforms is your data and its security. There's a reason they're free to use.
 
+The price of using Urbit is your own responsibility — you must run it yourself (or pay someone to run it for you).
 
-## Installing Urbit
+It's easy and it keeps getting easier.
 
-You'll want to install Urbit binary first.  Follow the instructions below to install the binary on Windows, MacOS, or Linux.
+There are currently two ways to run Urbit:
 
-### Windows
+If you use Mac or Linux and are not afraid of a few terminal commands, we recommend you run it yourself. That is the aim of this guide.
 
-> Please note that this method of installing Urbit is experimental, and we may not be able to assist you if you encounter issues related to WSL 2.
-
-Urbit cannot run on Windows itself, but there is a convenient way to run Linux using the [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) on Windows 10. Install the Windows Subsystem for Linux 2 and open a Linux terminal in Windows, then follow the Linux installation instructions below. These instructions have been tested and verified for WSL 2 + Ubuntu 18.04 LTS, as demonstrated in `~sitful-hatred`'s step-by-step setup guide [here](https://subject.network/posts/urbit-wsl2/).
+If you run Windows or just want to give it a shot without worrying about the gritty bits, you may want to consider a [hosting provider](/using/planet#hosting-providers).
```

Comment:
Er, technically no. I just think that because most people who run Windows are probably not up for the longer install process, we should flag it. It could be something like "If you run Windows and don't want to deal with the slightly longer install process or just want to give it a shot..." but the language gets muddy.